backend/authentication/enums.py

```diff
@@ -1,20 +1,23 @@
 # SPDX-License-Identifier: AGPL-3.0-or-later
 """
 Enums for the authentication app.
-More details about Enums can be found in schema section on Figma.
 """
 
 from enum import Enum
 
 
 class StatusTypes(Enum):
+    """Represents the possible statuses of a user."""

```

Comment:
Feel free to always do:

```
"""
Docstring even if it's a short one liner.
"""
```

The consistency makes it all a bit easier to digest and we like it when all the margins are aligned as happens when the docstring always starts on the next line 😊